### PR TITLE
Configures LocalExecutor using Python Buildpack

### PR DESCRIPTION
**Pull request by btylerburton**
Related to:
- https://github.com/GSA/data.gov/issues/4511

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,273 @@
...
+    # build: .
+    environment: &airflow-common-env
+        AIRFLOW__CORE__EXECUTOR: CeleryExecutor
+        AIRFLOW__DATABASE__SQL_ALCHEMY_CONN: postgresql+psycopg2://airflow:airflow@postgres/airflow
+        # For backward compatibility, with Airflow <2.3
+        ## ORIGINAL
+        # AIRFLOW__CORE__SQL_ALCHEMY_CONN: postgresql+psycopg2://airflow:airflow@postgres/airflow
+        ## ALTERED TO TEST
+        AIRFLOW__CORE__SQL_ALCHEMY_CONN: sql_alchemy_conn = postgresql+psycopg2://postgres:1234@localhost:5432/test
+        AIRFLOW__CELERY__RESULT_BACKEND: db+postgresql://airflow:airflow@postgres/airflow
```
Comment (jbrown-xentity) [inline docker-compose.yml]: Seems weird that we're not going to use celery with airflow, but we do in our local tests...

**Review by jbrown-xentity — DISMISSED** [review]
Looks good, though local dev and cloud.gov dev don't match (local docker uses celery)

File: docker-compose.yml (hunk 1 above)
Comment (robert-bryson) [reply docker-compose.yml]: I think this is a good point. We should test what we use and use what we test. 

It is likely the case that the `LocalExecutor` will not have the performance throughput that we need, but as far as I know that is yet to be verified. Tyler has made some progress with the inter-container communication and networking so I would like to spend a bit of time this afternoon experimenting with the `CeleryExecutor` again.

**Comment by btylerburton** [comment]
Based on what we've seen with the LocalExecutor, its almost certain that we want to push forward with Celery, but it does make sense to mirror local w/ remote, so I will make a default docker-compose using LocalExecutor and then include the other Celery docker setup as well.

File: docker-compose.yml (hunk 1 above)
Comment (btylerburton) [reply docker-compose.yml]: Just pushed up some changes to make LocalExecutor the default but also support Celery for our (eventual?) transition.

**Comment by btylerburton** [comment]
Just added a new docker-compose running LocalExecutor. We should be good to go here.

**Review by robert-bryson — APPROVED** [review]

**Pull request merged**